### DISCUSSION TITLE
Linux Window Decorations

### DISCUSSION THREAD

**Discussion by mahescho** [rustdesk/rustdesk#8669] [Q&A]
Some kind of own window decorations are used with Linux and not the system decorations of the window manager. Why? This causes a loss of functionality like moving to others desktops by the context menu when right clicking on the title bar.

**Comment by deuxdois** [comment]
Agreed on this. A very simple setting as is customary, to switch between own, native/system would be perfect.

This issue was closed with no comment, which makes me a little pessimistic about this ever getting relevant feedback, but let's see:
[https://github.com/rustdesk/rustdesk/issues/557](https://github.com/rustdesk/rustdesk/issues/557)

**Reply by mahescho** [reply]
I think this is due to "flutter": https://github.com/rustdesk/rustdesk/discussions/8791#discussioncomment-10166542

https://flutter.dev/
